pkg/router/metrics/haproxy/haproxy_test.go

```diff
@@ -22,108 +22,104 @@ func TestExporter_scrape(t *testing.T) {
-be_tcp:openshift-authentication:oauth-openshift,pod:oauth-openshift-5844b98b58-nm72j:oauth-openshift:port:10.129.0.42:6443,0,0,0,1,,3,20092,221613,,0,,0,0,0,0,UP,256,1,0,0,0,802,0,,1,16,1,,3,,2,0,,1,L4OK,,0,,,,,,,,,,,0,0,,,,,67,,,1,1,0,130,,,,Layer4 check passed,,2,3,4,,,,10.129.0.42:6443,,tcp,,,,,,,,0,3,0,,,0,,2,1,0,63237,
-be_tcp:openshift-authentication:oauth-openshift,pod:oauth-openshift-5844b98b58-z5rhm:oauth-openshift:port:10.130.64.11:6443,0,0,0,3,,13,7355,62957,,0,,0,0,0,0,UP,256,1,0,0,0,802,0,,1,16,2,,13,,2,0,,1,L4OK,,1,,,,,,,,,,,0,0,,,,,70,,,0,0,0,1493,,,,Layer4 check passed,,2,3,4,,,,10.130.64.11:6443,,tcp,,,,,,,,0,13,0,,,0,,0,1,0,59559,
-be_tcp:openshift-authentication:oauth-openshift,BACKEND,0,0,0,4,1,16,27447,284570,0,0,,0,0,0,0,UP,512,2,0,,0,802,0,,1,16,0,,16,,1,0,,1,,,,,,,,,,,,,,0,0,0,0,0,0,67,,,1,1,0,1615,,,,,,,,,,,,,,tcp,source,,,,,,,0,16,0,,,,,2,1,0,63237,
-be_secure:openshift-console:console,pod:console-6db7cbb464-gr787:console:port:10.129.0.43:8443,0,0,0,8,,241,505655,2344127,,0,,0,0,0,0,UP,256,1,0,0,0,802,0,,1,17,1,,0,,2,0,,57,L6OK,,1,5,226,1,4,0,0,,,,,0,0,,,,,11,,,0,0,2,1350,,,,Layer6 check passed,,2,3,4,,,,10.129.0.43:8443,7e4a3da6d0368ecb934a4910245f83b4,http,,,,,,,,0,15,221,,,0,,0,4,26,16533,
-be_secure:openshift-console:console,pod:console-6db7cbb464-8s44k:console:port:10.130.64.12:8443,0,0,0,0,,0,0,0,,0,,0,0,0,0,UP,256,1,0,0,0,802,0,,1,17,2,,0,,2,0,,0,L6OK,,1,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,Layer6 check passed,,2,3,4,,,,10.130.64.12:8443,5b10765dbf34d04f53986cf7ac1bf19c,http,,,,,,,,0,0,0,,,0,,0,0,0,0,
-be_secure:openshift-console:console,BACKEND,0,0,0,8,1,236,505655,2344127,0,0,,0,0,0,0,UP,512,2,0,,0,802,0,,1,17,0,,0,,1,0,,57,,,,5,226,1,4,0,0,,,,236,0,0,0,0,0,0,11,,,0,0,2,1350,,,,,,,,,,,,,1e2670d92730b515ce3a1bb65da45062,http,leastconn,,,,,,,0,15,221,0,0,,,0,4,26,16533,
-be_edge_http:openshift-console:downloads,pod:downloads-564948bf9c-vn6lh:downloads:port:10.128.0.30:8080,0,0,0,0,,0,0,0,,0,,0,0,0,0,UP,256,1,0,0,0,802,0,,1,18,1,,0,,2,0,,0,L4OK,,0,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,Layer4 check passed,,2,3,4,,,,10.128.0.30:8080,ce739475136fa468d51cfcf5aad91b68,http,,,,,,,,0,0,0,,,0,,0,0,0,0,
-be_edge_http:openshift-console:downloads,pod:downloads-564948bf9c-g7nsm:downloads:port:10.129.5.61:8080,0,0,0,0,,0,0,0,,0,,0,0,0,0,UP,256,1,0,0,0,802,0,,1,18,2,,0,,2,0,,0,L4OK,,0,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,Layer4 check passed,,2,3,4,,,,10.129.5.61:8080,450630300ddc04605decdd966ea57de6,http,,,,,,,,0,0,0,,,0,,0,0,0,0,
-be_edge_http:openshift-console:downloads,BACKEND,0,0,0,0,1,0,0,0,0,0,,0,0,0,0,UP,512,2,0,,0,802,0,,1,18,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,0,0,0,0,0,0,0,-1,,,0,0,0,0,,,,,,,,,,,,,a663438294fbd72a8e16964e97c8ecde,http,leastconn,,,,,,,0,0,0,0,0,,,0,0,0,0,
+		`public,FRONTEND,,,0,2,20000,162,18770,30715,0,0,0,,,,,OPEN,,,,,,,,,1,2,0,,,,0,0,0,1,,,,0,160,1,0,1,0,,0,1,162,,,0,0,0,0,,,,,,,,,,,,,,,,,,,,,http,,0,1,162,160,0,0,0,,,0,0,,,,,,,,,,,,,
```

Comment:
@frobware were the comments that you removed incorrect?

Reply:
Which comments?

Reply:
There was one on L44.  It should show up in the diffs.

Reply:
Also L64-66, 86, and 106-108.

Reply:
OK, will resurrect.

Reply:
Resolved in [this commit](https://github.com/openshift/router/pull/626/commits/fe66dcc8d4c68eea6ad735fa5072e4f5c562ca8b). The GitHub diff didn't display any comment removal, which I missed as I don't use the side-by-side view. It became clearer after reviewing the screenshot you provided.